latest_migrations.manifest

```diff
@@ -5,7 +5,7 @@ contenttypes: 0002_remove_content_type_name
 ee: 0015_add_verified_properties
 otp_static: 0002_throttling
 otp_totp: 0002_auto_20190420_0723
-posthog: 0364_team_external_data_workspace_rows
+posthog: 0365_pytz_update
```

Comment:
Had to update a bunch of packages to get things to work together. I don't like this migration though